projects/fal/src/fal/_serialization.py

```diff
@@ -76,26 +32,20 @@ def add_serialization_listeners_for(obj):
 
     if "." in module_name:
         package_name, *_ = module_name.partition(".")
-        _PACKAGES.add(package_name)
+        cloudpickle.register_pickle_by_value(__import__(package_name))
```

Comment:
this seems kinda weird/obscene that they actually require full modules to be there instead of the strings when they are comparing it against strings.

https://github.com/cloudpipe/cloudpickle/blob/d003266b18336e1e603536bdbe6518bc2dcc00d3/cloudpickle/cloudpickle.py#L181-L192

looking here it seems like they handle packages but i think it might be a good idea to make sure it works well with complex package structures